### PR TITLE
Database initialization

### PR DESCRIPTION
**Pull request by MDziugas**
Modified AppConfig to connect to DB

### CODE REVIEW

File: CSE/CSE/Program.cs [hunk 1]
```diff
@@ -2,17 +2,34 @@
...
+		/// The main entry point for the application.
+		/// </summary>
+		[STAThread]
+		static void Main()
+		{
+			System.Data.SqlClient.SqlConnection con = new System.Data.SqlClient.SqlConnection();
+			con.ConnectionString = ConfigurationManager.ConnectionStrings["MyDatabase"].ConnectionString;
+				
+			con.Open();
+			SqlCommand cmd = new SqlCommand("Select * From Table_1 ", con);
```
Comment (giedriusl) [inline CSE/CSE/Program.cs]: If this is for testing, it should be commented.

Comment (MDziugas) [reply CSE/CSE/Program.cs]: Fair enough

**Review by giedriusl — APPROVED** [review]

File: CSE/Logic/DataManagement/TextProcessing.cs [hunk 2]
```diff
@@ -4,11 +4,28 @@
...
 namespace Logic.ImageAnalysis
 {
-    class TextProcessing
+    public class TextProcessing
     {
+		public IEnumerable<string> _productDB;
+
+		public IEnumerable<string> ProductDB //Lazy initialization
+		{
+			get
```
Comment (giedriusl) [inline CSE/Logic/DataManagement/TextProcessing.cs]: Shouldn't this get with if be in Logic/Database/Reader.cs file like a separate method

File: CSE/Logic/DataManagement/TextProcessing.cs [hunk 3]
```diff
@@ -4,11 +4,28 @@
...
+			{
```
Comment (giedriusl) [inline CSE/Logic/DataManagement/TextProcessing.cs]: Shouldn't this get with if be in Logic/Database/Reader.cs file like a separate method

Comment (MDziugas) [reply CSE/Logic/DataManagement/TextProcessing.cs]: Spot on, will create a separate class for data models such as the latter

**Review by giedriusl — APPROVED** [review]

**Pull request merged**